docs/cosmos-hub/upgrade-node.md

```diff
@@ -0,0 +1,99 @@
+# Upgrade Your Node
```

Comment:
I think this guide needs a rethink and shouldn't block the rest of this PR. This should be a doc that describes the process for upgrading a network generically and explains each of the steps. This guide kinda jumps in at the mainnet upgrade and then has that content in less detail at the bottom of the doc.